content/blog/2017/2017-02-06-declarative-pipeline-ga.adoc

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "Declarative Pipeline Syntax 1.0 is GA"
+tags:
+- pipeline
+- blueocean
+author: hrmpw
+---
+[NOTE]
+====
+This is a guest post by
+link:https://github.com/HRMPW[Patrick Wolf],
+Director of Product Management at
+link:http://cloudbees.com[CloudBees]
+and contributor to
+the Jenkins project.
+====
+
+I am very excited to announce the addition of
+link:https://plugins.jenkins.io/pipeline-model-definition[Declarative Pipeline syntax]
+1.0 to
+link:https://plugins.jenkins.io/workflow-aggregator[Jenkins Pipeline.]
+We think this new syntax will enable everyone involved in DevOps, regardless of expertise,
+to participate in the continuous delivery process. Whether creating, editing or reviewing
+a pipeline, having a straightforward structure helps to understand and predict the
+flow of the pipeline and provides a common foundation across all pipelines.
+
+= Pipeline as Code
+Pipeline as Code was one of the pillars of the Jenkins 2.0 release and an essential part of implementing continuous delivery. Defining all of the stages of an application's CD pipeline within a "Jenkinsfile" and treating it as part of the application code automatically provides all of the benefits inherent in SCM:
+
+* Retain history of all changes to Pipeline
+* Rollback to a previous pipeline version
+* Review new changes to the pipeline in code review
+* Test new pipeline steps in branches
+* Audit changes to the pipeline
+* Run the same pipeline on a different Jenkins server
+
+= Getting Started with Declarative Pipeline
+We recommend people begin using it for all their Pipeline definitions in Jenkins.
+The plugin has been available for use and testing starting with the 0.1 release that was debuted at
+link:https://www.cloudbees.com/introducing-new-way-define-jenkins-pipelines[Jenkins World]
+in September, it has already been installed over 5,000 times.
```

Comment:
I'd end the sentence where the comma is. Then: "Since then, it has already been…"